### ISSUE TITLE
Update Grails Forge to Include more versions

### ISSUE THREAD

**Issue by jamesfredley** [grails/grails-forge#380]
Should be able to include next minor version, last minor version and potentially the next major version.  

It should still default to the current release.

See:  https://start.spring.io/

Version feed: https://start.grails.org/grails-version-feed.json 

start.grails.org gets snapshot from https://snapshot.grails.org/versions

it attempts to call the following, but they have not been deployed like snapshot.

https://next.grails.org/versions
https://prev.grails.org/versions

also add https://prev_snapshot.grails.org/versions

**Comment by jamesfredley** [comment]
related to https://github.com/grails/grails-forge-ui/issues/44

**Comment by jamesfredley** [comment]
This work has started in preparation for Grails 7.0.0.M1

start.grails.org already checks next.grails.org, prev.grails.org. prev_snapshot.grails.org would need to be added to forge-ui.

I believe we will end up creating three addition github workflows for next, prev and prev-snapshot.  We will also need to determine whether next, prev, prev-snapshot and snapshot are automatically triggered or manually run.  snapshot is currently automatically triggered, but this means that 6.2.2-SNAPSHOT and 7.0.0-SNAPSHOT are competing when PRs are merged into either branch.

The current GitHub actions are release and snapshot:

https://github.com/grails/grails-forge/blob/7.0.x/.github/workflows/snapshot.yml#L46

https://github.com/grails/grails-forge/blob/7.0.x/.github/workflows/release.yml#L108

Once there are 5 environments, here are initial thoughts on what should be deployed on each:

These are not labelled on start.grails.org, so keeping them as distinct versions is the goal since snapshot and next can be somewhat overlapping depending on current development work

initial thoughts

**before 7.0.0.M1 release:**
release: 6.2.x - this is also what IntelliJ uses in New Grails Project and is default, it should be the latest release version
snapshot: 6.2.x-SNAPSHOT
next: 7.0.x-SNAPSHOT initially
prev: 6.1.x-SNAPSHOT
prev-snapshot: 6.0.x-SNAPSHOT

**after 7.0.0.M1 release:**
release: 6.2.x
snapshot: 7.0.0.Mx
next: 7.0.x-SNAPSHOT
prev: 6.2.x-SNAPSHOT
prev-snapshot: 6.1.x-SNAPSHOT

**when 7.0.0 is released:**
release: 7.0.x
prev: 6.2.x
next: 7.0.x-SNAPSHOT
snapshot: 6.2.x-SNAPSHOT
prev_snapshot: 6.1.x-SNAPSHOT

**when 7.1.0-SNAPSHOT is started:**
release: 7.0.x
prev: 6.2.x
next: 7.1.x-SNAPSHOT
snapshot: 7.0.x-SNAPSHOT
prev-snapshot: 6.2.x-SNAPSHOT

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/pull/446

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/pull/450 https://github.com/grails/grails-forge/pull/451

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/pull/454

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/pull/456 - prev-snapshot workflow

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/pull/458

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/pull/459

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/pull/460

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge-ui/pull/45 - add prev-snapshot to version list on [start.grails.org](start.grails.org)

**Comment by jamesfredley** [comment]
Waiting on one issue with deployment on https://prev-snapshot.grails.org/versions before completing.

**Comment by jamesfredley** [comment]
5 versions on https://start.grails.org/
